### PR TITLE
chore: remove no job in queue log

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve? 

as title

### What is changed and how it works?

* only print log when enable failpoint
* do not truncate log file when run test case
* use go scripts to send feishu notify when test failed

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: tests/_utils/test_prepare [hunk 1]
```diff
@@ -144,9 +144,9 @@ function run_case() {
 
     eval ${init_table_cmd}
 
-    truncate -s 0 $WORK_DIR/master/log/dm-master.log
-    truncate -s 0 $WORK_DIR/worker1/log/dm-worker.log
-    truncate -s 0 $WORK_DIR/worker2/log/dm-worker.log
+    # truncate -s 0 $WORK_DIR/master/log/dm-master.log
```
Comment (GMHDBJD) [inline tests/_utils/test_prepare:147]: This is not a good way. Some tests rely on logs(check_log_contains). In shardddl tests, we run different cases in a single cluster, if we keep previous logs, check_log_contains is imprecise.

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**